### ISSUE TITLE
Inverted Map

### ISSUE THREAD

**Issue by Annyv2** [tum-vision/lsd_slam#126]
Hi! I have been testing the LSD_SLAM On an Ardrone 2.0, and I had to lower the resolution for it to work fast enough so it could keep tracking, but i'm obaining an inverted map as referenced here #122 , it can be seen in the picture. What's causing this and how can I make it work correctly? Thanks in advance!
![400288_2](https://cloud.githubusercontent.com/assets/5016479/8748601/647c390e-2c65-11e5-8794-dcb1c1146953.png) #

**Comment by v0n0** [comment]
Did you solve it?

**Comment by Annyv2** [comment]
No, @v0n0. I ended up testing with the image like that since it didn't affect a lot what I needed this for. Sorry I can't help you further. If you find a solution let me know :D
